### PR TITLE
Add code to allow the use of binder/start

### PR DESCRIPTION
**Pull request by scollis**
This PR may need some editing but gives some base code to allow those who build pangeo binders to use the base binder images as a starter AND use binder/start at the same time.

### CODE REVIEW

**Comment by scottyhq** [comment]
Thanks @scollis, this will be really useful for many folks! Is this now working for you if you build an image locally?

**Comment by scollis** [comment]
Hey @scottyhq I learnt a lot since this pr about the Pangeo build system. I need to modify. I’ve been unwell but will be back to this soon . It’s been a great learning experience about docker, repo2docker and binder 

-sent from a mobile device-

**Comment by yuvipanda** [comment]
@scollis I made a PR to this PR at https://github.com/scollis/pangeo-stacks/pull/1 that works for me locally :)

**Comment by scollis** [comment]
Awesome .. will give it a test

-sent from a mobile device-

File: onbuild/r2d_overlay.py [hunk 1]
```diff
@@ -109,6 +109,31 @@ def build():
...
+                )
+
+@become(NB_UID)
+def apply_start():
+    st_path = binder_path('start')
+
+    if os.path.exists(st_path):
+        return [
+            f'chmod +x {st_path}',
+            # since pb_path is a fully qualified path, no need to add a ./
```
Comment (jhamman) [inline onbuild/r2d_overlay.py]:
```suggestion
            # since st_path is a fully qualified path, no need to add a ./
```

**Comment by yuvipanda** [comment]
This LGTM but I don't have merge rights here

**Pull request merged**